[2022-07-22 13:00:26,203] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-22 13:00:26,249] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-22 13:00:26,250] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:26,254] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:26,255] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:26,318] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-07-04 00:00:00+00:00
[2022-07-22 13:00:26,325] {standard_task_runner.py:52} INFO - Started process 833 to run task
[2022-07-22 13:00:26,354] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-07-04T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmppqu7i96n', '--error-file', '/tmp/tmp1s4ztj77']
[2022-07-22 13:00:26,363] {standard_task_runner.py:80} INFO - Job 63: Subtask t1
[2022-07-22 13:00:26,443] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:26,826] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:27,067] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T00:00:00+00:00
[2022-07-22 13:00:27,069] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-22 13:00:27,185] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220704T000000, start_date=20220722T130026, end_date=20220722T130027
[2022-07-22 13:00:27,259] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:27,381] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:09,183] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-25 09:18:09,232] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [queued]>
[2022-07-25 09:18:09,233] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:09,236] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:09,237] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:09,288] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-07-04 00:00:00+00:00
[2022-07-25 09:18:09,298] {standard_task_runner.py:52} INFO - Started process 795 to run task
[2022-07-25 09:18:09,339] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-07-04T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp74v46t1_', '--error-file', '/tmp/tmp534l1xsr']
[2022-07-25 09:18:09,344] {standard_task_runner.py:80} INFO - Job 64: Subtask t1
[2022-07-25 09:18:09,380] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:09,474] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-07-04T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:09,758] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T00:00:00+00:00
[2022-07-25 09:18:09,761] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-25 09:18:09,895] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220704T000000, start_date=20220725T091809, end_date=20220725T091809
[2022-07-25 09:18:09,966] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:10,308] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
